runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/ProsSection/ProsSection.module.scss

```diff
@@ -0,0 +1,82 @@
+@import '#src-landing/scss/main.scss';
+
+$MIN-CARD-WIDTH: 280px;
+$ONE-COLUMN-BREAKPOINT: 600px;
+
+.root {
+  composes: root from '#src-landing/components/SectionContent/SectionContent.module.scss';
+  display: grid;
+  height: max-content;
+  grid-gap: 27px 23px;
+  grid-template-columns: repeat(4, 1fr);
+  grid-auto-flow: dense;
+  grid-auto-rows: minmax(80px, auto);
+}
+
+.box {
+  font-size: 150%;
+}
+
+.head {
+  padding: 5px;
+}
+
+.grid {
+  &__item {
+    composes: box;
+    max-width: 100%;
+    padding: 20px 20%;
```

Comment:
same here, I think that %, in this case, is not clear about its behaviour (at least for me)

Reply:
i can add a conment, but it looks nice so I would leave it like that